### PR TITLE
GH-342 add cordova-cli@8.1.1 hotfix blog post

### PR DESCRIPTION
**Pull request by brodybits**

### CODE REVIEW

File: www/_posts/2018-09-27-cordova-cli-8.1.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags: release tools
+---
+
+We are happy to announce that Cordova CLI `8.1.1` hotfix was released in September 2018.
+
+## Release Highlights
+
+The purpose of this hotfix release is to resolve a couple items that were broken in the Cordova CLI `8.1.0` release:
+
+* [GH-339](https://github.com/apache/cordova-cli/issues/339) Fix bin/cordova on Node.js 4
```
Comment (janpio) [inline www/_posts/2018-09-27-cordova-cli-8.1.1.md]: links are to issues, but text seems to be from PRs/commits?

Comment (brodybits) [reply www/_posts/2018-09-27-cordova-cli-8.1.1.md]: Yes. There was one PR with 2 commits to resolve the 2 issues. I can resolve this in one of two ways:
1. Change GH numbers to the PR number and change above text to be more consistent with this solution
2. Change the text to reflect the issue actually solved

I would probably favor the first one. I think it is better for the release note to tell what was actually changed, especially in a hot fix.

**Comment by brodybits** [comment]
@janpio I just made another squash fix for better consistency with the change descriptions.

The one minor downside is that we are using the same GH number for all (both) changes. But I think it is worth it for nicer consistency.

**Comment by janpio** [comment]
I now see GH-X linking to an issue Y - that doesn't make sense.

Just add both links somehow and we will define the actual, useful format in the future for the next releases.

**Comment by brodybits** [comment]
I just reverted the last change, gonna squash-merge it now.

**Pull request merged**